bin/reth/Cargo.toml

```diff
@@ -119,6 +119,20 @@ min-warn-logs = ["tracing/release_max_level_warn"]
 min-info-logs = ["tracing/release_max_level_info"]
 min-debug-logs = ["tracing/release_max_level_debug"]
 min-trace-logs = ["tracing/release_max_level_trace"]
+optimism = [
+  "reth-primitives/optimism",
+  "reth-revm/optimism",
+  "reth-interfaces/optimism",
+  "reth-rpc/optimism",
+  "reth-rpc-engine-api/optimism",
+  "reth-transaction-pool/optimism",
+  "reth-provider/optimism",
+  "reth-beacon-consensus/optimism",
+  "reth-basic-payload-builder/optimism",
+  "reth-network/optimism",
+  "reth-network-api/optimism"
+]
+ethereum = []
```

Comment:
I don't think we need this?
what is this used for?

Reply:
This is a no-op feature, specifically for the CI matrix. We added the `ethereum` feature as a switch for CI with and without `optimism` enabled.

Reply:
I see, makes sense imo

let's add s one line note